hazelcast-spring/src/test/resources/com/hazelcast/spring/fullcacheconfig-applicationContext-hazelcast.xml

```diff
@@ -49,7 +49,7 @@
                 <hz:property name="hazelcast.merge.first.run.delay.seconds">5</hz:property>
                 <hz:property name="hazelcast.merge.next.run.delay.seconds">5</hz:property>
             </hz:properties>
-            <hz:wan-replication name="testWan">
+            <hz:wan-replication name="testWan" snapshot-enabled="false">
                 <hz:target-cluster group-name="tokyo" group-password="tokyo-pass">
                     <hz:replication-impl>com.hazelcast.wan.impl.WanNoDelayReplication</hz:replication-impl>
```

Comment:
I think such config should be blocked via fail fast approach. We should not allow both OSS Publisher ```WanNoDelayReplication``` and ```snapshot-enabled``` at the same time.